jupyterhub/apihandlers/shares.py

```diff
@@ -95,7 +96,7 @@ def server_model(self, spawner):
             }
         }
         # subset keys for sharing
-        for key in ["name", "url", "ready"]:
+        for key in ["name", "url", "full_url", "ready"]:
```

Comment:
The function's docstring mentions the elements in this list and has now become outdated

Reply:
thanks, added.